packages/datatrak-web-server/src/routes/SurveyRoute.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { Request } from 'express';
+import camelcaseKeys from 'camelcase-keys';
+import { Route } from '@tupaia/server-boilerplate';
+import { DatatrakWebSurveysRequest } from '@tupaia/types';
+
+export type SurveyRequest = Request<
+  DatatrakWebSurveysRequest.Params,
+  DatatrakWebSurveysRequest.ResBody[number],
+  DatatrakWebSurveysRequest.ReqBody,
+  DatatrakWebSurveysRequest.ReqQuery
+>;
+
+export class SurveyRoute extends Route<SurveyRequest> {
+  public async buildResponse() {
+    const {
+      ctx,
+      query = {},
+      params: { surveyCode },
+    } = this.req;
+    const { fields } = query;
+    const surveys = await ctx.services.central.fetchResources('surveys', {
```

Comment:
Any reason that you went for the filter over the url param? (eg. /surveys/:recordId)